doc/contribute/build_ccf.rst

```diff
@@ -59,7 +59,19 @@ Tests can be started through the ``tests.sh`` wrapper for ``ctest``.
 .. code-block:: bash
 
     $ cd build
-    $ ./tests.sh
+    $ ./tests.sh -LE partitions
+
+The `partitions` test requires `root` access (to set up firewall rules), so it will not pass when using normal user accounts.
+To run the partitions test, please use `sudo`.
+
+.. code-block:: bash
+
+    $ sudo ./tests.sh -R partitions
+
+Be sure to run the non-root tests first, or the configuration will be created as root and break some non-root tests from altering it.
+
+.. note::
+   Use the `-j<threads>` option to speed up tests, unless you're running benchmarks, in which case it's best to run single-threaded.
```

Comment:
This definitely nitpicking, but I still take exception to the idea that we document features of 3party tooling. Documentation is expensive to produce and to maintain, I think it should cover all of CCF, and nothing but CCF.